101-powerbi-workspace-create/azuredeploy.json

```diff
@@ -0,0 +1,51 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "powerbiWorkspaceName": {
+      "type": "string",      
+      "minLength": 3,
+      "maxLength": 63,
+      "metadata": {
+        "description": "the name of the PowerBI workspace collection name. It should match with the following regular expression: ^(?:[a-zA-Z0-9]+-?)+$ or it will raise an error. "
+      }
+    },
+    "powerbiWorkSpaceLocation":{
+      "type": "string",
+      "metadata": {
+        "description":"the location to create the powerbi workspace collection. At present this resource is available in southcentralus,northcentralus,eastus2,westus,westeurope,northeurope,brazilsouth,southeastasia,australiasoutheast,canadacentral,japaneast,uksouth,westindia locations"
```

Comment:
remove the list of locations since this will quickly become obsolete - we can send folks to a link if we have one